.github/workflows/ci.yml

```diff
@@ -196,7 +196,7 @@ jobs:
           kubectl apply -f config/samples/simple.yaml
         shell: bash
       - name: Check and wait pulp-operator deploy
-        run: kubectl wait --for condition=Pulp-Operator-Finished-Execution pulp/example-pulp --timeout=600s
+        run: kubectl wait --for condition=Pulp-Operator-Finished-Execution pulp/example-pulp --timeout=900s
```

Comment:
increasing timeout because django migrations are taking longer with the newer pulpcore versions